CAP/gap/ConstructiveCategoriesRecord.gi

```diff
@@ -73,7 +73,17 @@ IsSymmetricClosedMonoidalCategory  := Concatenation( [
 IsRigidSymmetricClosedMonoidalCategory  := Concatenation( [ 
 "TensorProductInternalHomCompatibilityMorphismInverseWithGivenObjects",
 "MorphismFromBidualWithGivenBidual"
-], ~.IsSymmetricClosedMonoidalCategory )
+], ~.IsSymmetricClosedMonoidalCategory ),
+
+IsAbelianCategoryWithEnoughProjectives := Concatenation( [
+"EpimorphismFromSomeProjectiveObject",
+"ProjectiveLift"
+], ~.IsAbelianCategory ),
+
+IsAbelianCategoryWithEnoughInjectives := Concatenation( [
+"MonomorphismIntoSomeInjectiveObject",
+"InjectiveColift"
```

Comment:
Now the object functions are gone? It this on purpose?

Reply:
They are not necessary since they can be derived from
`MonomorphismIntoSomeInjectiveObject`
and
`EpimorphismFromSomeProjectiveObject`.
If I remember it correctly, then the record
`CAP_INTERNAL_CONSTRUCTIVE_CATEGORIES_RECORD`
was intended to point out only one particular way to implement
a given type of category, and not to enlist all possible functions
for such a type.

Reply:
I think you are right, yes.